### PR TITLE
Add Unicorn

### PR DESCRIPTION
**Pull request by MrBrain295**
Add the "Unicorn" programing language. I got the code from [here](http://btiffin.users.sourceforge.net/up/unicon.html)

### CODE REVIEW

File: u/Unicorn.icn [hunk 1]
```diff
@@ -0,0 +1,3 @@
+procedure main()
+    write("Hello, world")
```
Comment (Richienb) [inline u/Unicorn.icn:2]:
```suggestion
    write("Hello World")
```

**Review by Richienb — CHANGES_REQUESTED** [review]
Remember to update the readme

**Comment by MrBrain295** [comment]
fixes part of: #790

File: u/Unicorn.icn (hunk 1 above)
Comment (MrBrain295) [reply u/Unicorn.icn]: Done

**Comment by MrBrain295** [comment]
Thanks

**Review by frasca617 — COMMENTED** [review]
Thank you

**Pull request merged**